test/common/upstream/cluster_manager_impl_test.cc

```diff
@@ -916,6 +917,7 @@ version_info: version3
 static_clusters:
   - cluster:
       name: "cds_cluster"
+      type: "STATIC"
```

Comment:
Did you just add this back for clarity? Or was this required for some reason? If it was required we might need to think about whether this is a breaking change or not.

Reply:
This is needed only for this test, because this test is comparing the original yaml vs the result of  serialized protobuf object.  The fact that type: "STATIC" got split out here is evident that yaml without explicitly setting type is correctly serialized to STATIC as before.  Also we have other tests in here that doesn't specify the type.

Reply:
:+1: